### ISSUE TITLE
Development builds no longer available

### ISSUE THREAD

**Issue by wetneb** [gephi/gephi#2830]
The links to the various development builds in the README.md no longer work, because of some change in Maven Central.
In OpenRefine we have the same problem (because we shamelessly copied the pipeline you use for development builds).

I had a conversation with the support staff of the repository and you can find it here: https://github.com/OpenRefine/OpenRefine/issues/5947#issuecomment-1617325426

TLDR: this usage of the repository does not really seem to be supported, so in OpenRefine we are considering migrating to something else.
